Dockerfile

```diff
@@ -65,26 +33,53 @@ RUN --mount=type=cache,target=/root/.cache/go-build \
     for f in $(find bin -executable -type f); do xx-verify $f; done
 
 FROM --platform=${BUILDPLATFORM} scratch AS spire-base
+COPY --link --from=builder --chown=root:root --chmod=755 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 WORKDIR /opt/spire
```

Comment:
As discussed offline, this could have some unintended consequences for users who run the containers with non-default commands that rely on relative paths from `/opt/spire`. I'm now convinced we should keep this directive as is.